README.md

```diff
@@ -1,225 +1,123 @@
+[![Contributors][contributors-shield]][contributors-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+
+
+
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/DidierRLopes/GamestonkTerminal">
+    <img src="images/logo.png" alt="Logo" width="80" height="80">
+  </a>
+
+  <h3 align="center">Gamestonk Terminal 🚀</h3>
+
+  <p align="center">
+    The next best thing after Bloomberg Terminal.
+    <br />
+    <a href="https://github.com/DidierRLopes/GamestonkTerminal/FEATURES.md"><strong>Features »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/DidierRLopes/GamestonkTerminal/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5BBug%5D">Report Bug</a>
+    ·
+    <a href="https://github.com/DidierRLopes/GamestonkTerminal/issues/new?assignees=&labels=enhancement&template=enhancement.md&title=%5BIMPROVE%5D">Suggest Improvement</a>
+    ·
+    <a href="https://github.com/DidierRLopes/GamestonkTerminal/issues/new?assignees=&labels=new+feature&template=feature_request.md&title=%5BFR%5D">Request Feature</a>
+  </p>
+</p>
+
+
+
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary><h2 style="display: inline-block">Table of Contents</h2></summary>
+  <ol>
+    <li> <a href="#about-the-project">About The Project</a> </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#install">Install</a></li>
+        <li><a href="#api-keys">API Keys</a></li>
+        <li><a href="#usage">Usage</a></li>
+      </ul>
+    </li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#disclaimer">Disclaimer</a></li>
+    <li><a href="#contact">Contact</a></li>
+    <li><a href="#acknowledgments">Acknowledgments</a></li>
+  </ol>
+</details>
+
+## About The Project
+
+How it started:
+"Gamestonk Terminal is an awesome stock and crypto market terminal that has been developed for fun, while I saw my GME shares tanking. But hey, I like the stock 💎🙌."
+
+How it's going:
+u/schoen: _(...) I think people have upvoted this so much because it's an initial version of a single person's passion project that manages to put a whole bunch of financial information at your fingertips. With an active community, it could easily grow and grow and grow in functionality._
+
+
+## Getting Started
+### Install
+
+Supported Python versions: 3.6, 3.7, 3.8
+
+1. Install [Anaconda](https://docs.anaconda.com/anaconda/install/)
+
+Confirm that you have it with: `conda -V`
+The output should be like `conda 4.9.2`
+
+2. Create Environment
+
+You can name the environment whatever you want. Although you could use names such as: `welikethestock`, `thisistheway` or `diamondhands`, we recommend something simple and intuitive like `gst`. This is because this name will be used from now onwards.
+```
+conda create -n gst python=3.6.8
+````
 
-## About
+3. Fork the Project
```

Comment:
We should put all of the contributing stuff into a `CONTRIBUTORS.md` file instead.